src/main/java/com/example/naitei19javaecommerce/controller/admin/InvoiceController.java

```diff
@@ -43,4 +45,13 @@ public String getPaymentHistory(Model model,
         model.addAttribute("payments", paymentHistoryResponseList);
         return "admin/invoice/index";
     }
+
+    @GetMapping("/new")
+    public String getNewOrder(Model model) {
```

Comment:
```suggestion
    @GetMapping("/index/waiting")
    public String ordersByStatus(Model model) {
```
Sau em đưa waiting là 1 tham số đi
để sau ko chỉ cố định là waiting mà chị muốn action theo các status thì nó khỏi tốn thêm action mới